lib/active_campaign/v3/client.rb

```diff
@@ -0,0 +1,60 @@
+require 'httparty'
```

Comment:
Why `HTTParty` instead of just sticking with the other `HTTP` gem we were using?

Reply:
There was something I ran into yesterday that I was having a hard time getting HTTP.rb to do and their docs don't seem very thorough.

It may have had to do with being able to change the `base_uri` at any point through options passed to the request, which would help for those endpoints we discussed yesterday that were different from all the others, but I can't remember for sure if that was the only thing.

Are you opposed to HTTParty? From what I've seen, it seems to have a broader user base, which makes it easier when trying to figure out how to make it do something.

Reply:
Opposed? No. Just curious about rationale. Does that mean we'll use it for the V2 bits as well?

Reply:
Already are.

Reply:
...as of #8...not merged yet.